### PR TITLE
Add docker for client test

### PR DESCRIPTION
**Pull request by penpyt**
add kelpie and kelpie-test client in docker to easy testing in kubernetes

### CODE REVIEW

File: client-test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8u212-jre-slim-stretch
+ENV KELPIE_VERSION 1.1.0
+
+RUN apt-get update && apt-get install wget unzip -y && \
```
Comment (feeblefakie) [inline client-test/Dockerfile:4]: Let's use gradle shadowJar.
I think the following should work in build.gradle.
```
docker {
    files tasks.shadowJar.outputs
}
```
Then, Dockerfile can get all the jars including kelpie.

File: client-test/Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (feeblefakie) [inline client-test/Makefile]: I don't think we need this by using docker gradle plugin and shadowJar.
I also think pushing should be done separately from building.

File: client-test/build.gradle [hunk 3]
```diff
@@ -8,12 +8,17 @@ repositories {
...
 
 dependencies {
-    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.0.0'
+    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.1.0'
     implementation group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.0.5'
     implementation group: 'com.scalar-labs', name: 'scalardb', version: '2.0.1'
     implementation group: "io.github.resilience4j", name: "resilience4j-retry", version: "1.3.1"
 }
 
+task sourcesJar(type: Jar) {
```
Comment (feeblefakie) [inline client-test/build.gradle]: Is it needed ?

**Review by feeblefakie — CHANGES_REQUESTED** [review]
Left several comments.

File: client-test/build.gradle (hunk 3 above)
Comment (penpyt) [reply client-test/build.gradle]: removed

File: client-test/Makefile (hunk 2 above)
Comment (penpyt) [reply client-test/Makefile]: ok fixed in https://github.com/scalar-labs/kelpie-test/pull/2/commits/205579cd7434e36d825be604616d571d7c6d9bc5 and https://github.com/scalar-labs/kelpie-test/pull/2/commits/597c58a5e7d75aedbc45adcfdec8e21d86dcea8b

File: client-test/Dockerfile (hunk 1 above)
Comment (penpyt) [reply client-test/Dockerfile:4]: done, it works

**Review by feeblefakie — APPROVED** [review]
LGTM !

**Review by yito88 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**